data-mermaid/mermaid-webapp
# 376: Add methods filter collect page

[trello ticket](https://trello.com/c/wOlKJC46/376-add-the-method-filter-on-tables-from-v1)

Methods filter added to collect records page - multiselect drop down which works alongside global search

notes:
- using MUI component library for multiselect checkbox
- need to drill down further to get hoverstate matching theme color (will add a separate ticket)
- additional separate ticket added to update pagination counter so when filters are added (global and method drop down), table record count will be updated as well in the pagination counter
- added persist method filter

to test:
- go to collect records page table
- select items to filter in methods checkbox
- refresh, ensure filters are still checked and table is updated

intended behviour:
- all selected, show all records
- if none are selected, show all records (this is the default, which implies filter is inactive)


EDIT - working on fixing tests
